README.md

```diff
@@ -71,6 +71,39 @@ x1 rotate by 1 = (0.5, 0.75, 1, 2, 3, 4, 5, 0.25,  ... ); Estimated precision: 4
 x1 rotate by -2 = (4, 5, 0.25, 0.5, 0.75, 1, 2, 3,  ... ); Estimated precision: 43 bits
 ```
 
+### Memory issues
+OpenFHE is a memory-optimized C++ library, but these optimizations can cause
+memory issues when transitioning to Julia.
+
+In OpenFHE, large objects like `Ciphertext`, `Plaintext`, and `CryptoContext` are managed
+using `std::shared_ptr`. These objects are not freed until all associated `std::shared_ptr`s
+are destroyed. Although `std::shared_ptr`s are relatively small, Julia's garbage collector
+doesn't always free them automatically. This is because Julia's garbage collector primarily
```

Comment:
```suggestion
are destroyed. Since the Julia objects that hold a reference to these shared pointers are relatively small, Julia's garbage collector
does not always free them automatically, as they are not considered a high priority for garbage collection. This is because Julia's garbage collector primarily
```